consts.py

```diff
@@ -0,0 +1,11 @@
+POST = "POST"
+QUESTION = "question"
+ANSWER = "answer"
+CONTENT = "content"
+CARD_TYPE = "card_type"
+DATE = "date"
```

Comment:
General: Consider working from lower to upper layers, for instance, start with implementing the DB and the other classes used in the controller and afterwards connect them all with the controller. It'll make your work much easier and it won't leave "dead" code path (paths in code that do not work/do nothing). 
General: Try to prioritize your tasks, start with the most important tasks (for instance, inserting a card) and make sure it's finished. Only when it's done and ready, move to the next tasks with lower priority. It'll make your work more organized and again won't leave "dead" code paths.